modules/virt-creating-data-volumes-using-storage-api.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-data-volumes.adoc
+
+[id="virt-creating-data-volumes-using-storage-api_{context}"]
+
+= Creating data volumes using the Storage API
+
+When you create a data volume using the *storage* API, the CDI optimizes your allocation based on the attributes of the mounted volume. You only have to specify the data volume name, namespace, and the amount of storage you want to allocate.
+
+For example:
+
+* When using Ceph RBD, your access mode is automatically set to `ReadWriteMany`, which improves efficiency during Live Migration, among other performance benefits.
+* File system overhead is automatically calculated for you. If you request two gigabytes of storage, you get exactly two gigabytes of storage, as opposed to a lesser amount when you use the *pvc* API.
+
+In the following YAML, using the *storage* API allocates a data volume with a storage capacity of two gigabytes, as in the previous *pvc* API allocation example. In this case, however, you do not specify specify `accessModes` or `volumeMode`. Instead, these values are optimized and calculated for you by the CDI, based on the type of disk you mount or the defaults you define in your storage profile.
```

Comment:
You do not need to specify volumeMode or accessMode but you may.

Reply:
based on the type of storage or the defaults you define in your storage profile.

Reply:
How's this:

_In the following YAML, using the *storage* API allocates a data volume with a storage capacity of two gigabytes, as in the previous *pvc* API allocation example. In this case, it is not required to specify specify `accessModes` or `volumeMode`. These values are optimized and calculated for you by the CDI, based on the type of storage or the defaults you define in your storage profile. If you want to provide custom values, they will override the system-calculated values._